tests/remotes/ssh.py

```diff
@@ -15,6 +18,39 @@
 )
 
 
+class SSHMockHandler(Handler):
```

Comment:
See this for the reason, https://github.com/carletes/mock-ssh-server/issues/22

Reply:
@isidentical Please add a comment with a link, so we don't forget to remove it in the future.

Reply:
```suggestion
# See: https://github.com/carletes/mock-ssh-server/issues/22
class SSHMockHandler(Handler):
```